MapTeleport/MapTeleport.cs

```diff
@@ -40,7 +40,7 @@ protected override void Dispose(bool disposing)
 			if (player == null || !player.Active)
 				return;
 
-			player.SetData("mapteleport", false);
+			player.SetData("mapteleport", true);
```

Comment:
I'll change that so that you require a permission to automatically have maptp enabled when joning. Either way great job thx.